src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -159,6 +159,7 @@
       <mat-card-subtitle class="m-0">{{ selectedVersion?.name }}</mat-card-subtitle>
     </mat-card-header>
     <mat-card-content class="p-3">
+      <div><strong matTooltip="DOI listed in descriptor">DOI</strong>: {{ selectedVersion.doiURL ? selectedVersion.doiURL : 'n/a' }}</div>
```

Comment:
Agreed a screenshot would help visualize this...

What is `DOI listed in the descriptor`? Which descriptor? Should it just be `DOI`?

Also, more JavaScript-y way to write this would be `{{ selectedVersion.doURL || 'n/a' }}` (still not sure if that's a good TypeScript practice) :)

Reply:
changed matToolTip to "Digital Object Identifier"